Dockerfile-backend-nvidia

```diff
@@ -0,0 +1,21 @@
+FROM nvidia/cuda:12.0.0-cudnn8-runtime-ubuntu22.04
+
+# Lesser layers are always better but one or two more don't matter if it increases repeated build speed
+# Install package manager dependencies
+RUN apt-get update ; \
+    apt-get install -y --no-install-recommends git build-essential g++ libgomp1 git python3 python3-dev python3-pip
+
+WORKDIR /app
+COPY requirements.txt ./
+
+# Install pip requirements
+RUN pip install -r requirements.txt
+RUN pip install --force-reinstall --no-cache-dir hnswlib
+
+# Clone and install nomic
+RUN git clone https://github.com/nbdy/nomic ; cd nomic ; pip install .[GPT4All] ; pip install .
```

Comment:
Please note that this line should be changed as soon as the pull request [#150](https://github.com/nomic-ai/nomic/pull/150) in the nomic repository has been accepted.

Reply:
Fixed, thanks! https://github.com/Josh-XT/Agent-LLM/commit/80c1a5401ae41f7b52fd1007648d034b18c1c2cb